src/Home.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  color: #383854;
+  font-family: Righteous;
```

Comment:
I'm assuming you are using the same font-size throughout your app if this is the case it should probably be `injectGlobal` from another file, otherwise you will be adding it again and again.

Currently the nav bar has a different font.

Reply:
Another note on the nav bar is that it seems very small now you have added the rest of the content